ush/sub_acorn

```diff
@@ -0,0 +1,171 @@
+#!/bin/sh --login
```

Comment:
This is fine, but would linking `sub_wcoss2` as `sub_acorn` and replacing `wcoss2|acorn` with `machine` by calling `detect_machine.sh` be sufficient?

Reply:
Yes, but instead of sourcing `detect_machine.sh`, I decided to pass the machine name into `sub_*`.